Formula/snort.rb

```diff
@@ -55,7 +54,8 @@ def install
     system "make", "install"
 
     rm Dir[buildpath/"etc/Makefile*"]
-    (etc/"snort").install Dir[buildpath/"etc/*"]
+    (buildpath/"etc").children.each { |f| chmod 0644, f }
```

Comment:
Not sure what permissions it has but that should probably be reported

Reply:
Yup, working on it. Just wanted to check that it helps before reporting (and that's why I've added `do not merge` label)

Reply:
Have posted a question to snort-devel@lists.snort.org (as soon as it will be approved by a moderator will add a link)